packages/agent/conf/demos/dbc/presentation_definitions/SphereonWalletIdentityCredential.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
we already have a sphereonWalletIdentity.json, so what is the difference between this one and the one we already have?

Reply:
hmmm, what is the logic behind the conf folder for the agent? this places a dbc conf folder, which i think is fine. But then we should apply this logic to all demo's. And then the question becomes. how do we handle duplicates? i am not sure of managing duplicates is a nice approach.

Reply:
For some files like the dids we must have different files per ecosystem. Like with a deployment for knb/kvk we can't have the files from other ecosystems.
Also an agent running for banqup has no business running configs for knb etc.
Maintaining duplicates is not ideal in all cases, however in other cases you might need a differences in files between the ecosystems like a different PD text for the same credential. Isolating the configs per ecosystem also makes sure everything is testable for that specific ecosystem. It has everything it needs to run a deployment nothing more, nothing less.